gallery/src/components/NxNexusPageHeader/NexusPageHeaderExample.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {faSave, faCog } from '@fortawesome/free-solid-svg-icons';
+
+import { NxButton, NxNexusPageHeader, NxFontAwesomeIcon } from '@sonatype/react-shared-components';
+
+const NexusPageHeaderExample = () => {
```

Comment:
This (and the file itself) should be `NxNexusPageHeaderExample`

Reply:
> This (and the file itself) should be `NxNexusPageHeaderExample`

https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/65a03b62ed4d55bc121b821d98befabc0659788f